static/home/js/signprot-multitabtable.js

```diff
@@ -92,73 +92,158 @@ function reset_tab() {
     window.location.href = "/signprot/couplings";
 }
 
-// Draft for calculation of normalized rank for a given column
-// NOTE: the support is currently not taken into account
+// Calculation of normalized rank for a given column
 function createRank(table_id, column) {
```

Comment:
Two ideas for reducing the duplication in the code: 
1. store the table number in the span when you are  calling the createRank function (similar to the data-raw and data-column-nr). In that case you don't have to duplicate the filter and the initialization code. 

2. Use an in-between function that calls the filter with the table #:
<pre>
function rankedRangeFiltert1(filterVal, columnVal, rowValues, stateVal){
     return rankedRangeFilter(filterVal, columnVal, rowValues, stateVal, "1");
}

function rankedRangeFiltert2(filterVal, columnVal, rowValues, stateVal){
     return rankedRangeFilter(filterVal, columnVal, rowValues, stateVal, "2");
}

# And adjust the filter to use the extra "1" or "2" in the 
function rankedRangeFilter(filterVal, columnVal, rowValues, stateVal, tableNum){
     ... 
}
</pre>